pkgs/data/fonts/babelstone-han/default.nix

```diff
@@ -1,19 +1,19 @@
 {stdenv, fetchzip}:
 
 let
-  version = "11.0.0";
+  version = "11.0.2";
 in fetchzip {
   name = "babelstone-han-${version}";
 
-  url = http://www.babelstone.co.uk/Fonts/3902/BabelStoneHan.zip;
+  url = http://www.babelstone.co.uk/Fonts/Download/BabelStoneHan.zip;
```

Comment:
mhm. Is this url stable? It might break the checksum in future.

Reply:
> Is this url stable?
It was not stable, every version had different number in place of ```3902```.
Changing it to ```Download``` might indicate upstream's plans on its stabilization and possible hash breakages